lawnchair/src/app/lawnchair/ui/TouchTarget.kt

```diff
@@ -16,6 +16,7 @@
 
 package app.lawnchair.ui
 
+import android.widget.Button
```

Comment:
This is the _View_ Button class, it looks like the doc is referencing a _composable_ Button.

Reply:
949d54d2781bb47f2a8b1fcc9553d3ba43178933